js/scripts/main.js

```diff
@@ -48,7 +48,7 @@ spanishButton.addEventListener("click", () => {
 
 // (3) Set the current ZONE to be the first one we encounter - village.main.
 
-let ZONE = Zone.getZones().village.main;
+let ZONE = Zone.getZones().playground.nathan;
```

Comment:
We should be loading village.main, not playground.nathan. I think this should be changed before merging.